brave/brave-ios
# Fix #2401: Add disabled states for tips & monthly contributions

## Summary of Changes


This pull request fixes #2401

## Submitter Checklist:


## Test Plan:

- Open Rewards Settings
- Verify "View Details" button visible while rewards is enabled
- Verify image + label appear when rewards is disabled


## Screenshots:

![Simulator Screen Shot - iPhone 11 Pro - 2020-03-17 at 12 31 11](https://user-images.githubusercontent.com/529104/76878720-9af7fa00-684b-11ea-9526-a7c59c01eab6.png)

## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
